Dockerfile.no-wheelhouse

```diff
@@ -0,0 +1,42 @@
+ARG FROM_IMAGE=python:slim
+FROM $FROM_IMAGE
+LABEL maintainer "Praekelt.org <sre@praekelt.org>"
+
+# pip: Disable cache and use Praekelt.org Python Package Index
```

Comment:
Should probably update this comment to remove reference to praekelt python package index